frontend/src/pages/Admin/Components/AddTeamMember/AddTeamMember.jsx

```diff
@@ -0,0 +1,140 @@
+            <form className={styles["inside-add-team-member"]} onSubmit={onSubmit}>
+              <Grid container>
+                <Grid xs={12} sm={2} md={3}/>
+                <Grid xs={12} sm={8} md={6}>
+                  <div className={styles["upload-section"]} onClick={changePic}>
+                    <img
+                      src={picUrl}
+                      className={styles["img-admin"]}
+                      alt="admin_img"
+                    />
+                    <h1 className={styles["h1"]} style={{color:"white", fontSize:"15px"}}>Click to Change</h1>
+                    <input
+                      id="profile-pic-input"
+                      type="file"
+                      accept="image/*"
+                      capture="camera"
+                      onChange={onPicChange}
+                      required="required"
+                      style={{display:"none"}}
+                    />
+                  </div>
+                </Grid>
+              </Grid>
+              <div className={styles["add-team-member-input"]}>
+                <input
+                  id="txt_name"
+                  type="text"
+                  required="required"
+                  name="fullName"
+                  placeholder="Full Name"
+                />
+                <i className="fas fa-signature"></i>
+              </div>
+              <div className={styles["add-team-member-input"]}>
+                <input
+                  id="txt_description"
+                  type="text"
+                  required="required"
+                  name="description"
+                  placeholder="Short Description"
+                />
+                <i className="fas fa-prescription"></i>
+              </div>
+              <div className={styles["add-team-member-input"]}>
+                <input
+                  id="txt_linkedin"
+                  type="url"
+                  required="required"
+                  name="linkedin"
+                  placeholder="LinkedIn URL"
+                />
+                <i className="fab fa-linkedin-in"></i>
+              </div>
+              <div className={styles["add-team-member-input"]}>
+                <input
+                  id="txt_github"
+                  type="url"
+                  required="required"
+                  name="github"
+                  placeholder="Github URL"
+                />
+                <i className="fab fa-github"></i>
+              </div>
+              <div className={styles["add-team-member-input"]}>
+                <input
+                  id="txt_twitter"
+                  type="url"
+                  required="required"
+                  name="twitter"
+                  placeholder="Twitter URL"
+                />
+                <i className="fab fa-twitter"></i>
+              </div>
+              <label className={styles["add-team-member-label-text"]}>Teams :</label>
```

Comment:
Let's keep it's color black/grey(same as input fields place-holder) In order to be cosistent